### ISSUE TITLE
Stopped working with 10.10.4 (14E26a)

### ISSUE THREAD

**Issue by mogilner** [d235j/GVFixer#5]
I have been using this fix for a long time and has been working great.

After installing the newest beta that came out today. The fix is no longer working. 

I am getting the "Unable to build JSON Object from JSON string error"

If you need anything from me to help troubleshoot this, please let me know.

I have reinstalled both Easy SIMBL and also the fix itself. I also know about making sure to check USE SIMBL after each update.

**Comment by mogilner** [comment]
Here is the error message I now get.
![screen shot 2015-05-26 at 3 18 34 pm](https://cloud.githubusercontent.com/assets/9003795/7822368/9c0f4dd0-03ba-11e5-9b26-c22420bc06ae.png)

**Comment by Fofer** [comment]
Newest beta of what? Mac OS X? What's the version number?

**Comment by jaw206** [comment]
I am getting the same 404 error on 10.10.3.

**Comment by d235j** [comment]
Looks like Google removed the API.
Well, it was fun while it lasted.

**Comment by Fofer** [comment]
Uh oh.

Strangely notable however, is that it's still working for me.  I'm on OS X 10.10.3. 

I just left myself a message, refreshed GrowlVoice (with GVFixer via EasySIMBL enabled, obviously) and the voicemail message appeared there.

**Comment by d235j** [comment]
Quit and reopen the app and it won't work anymore.

Sean Kovacs (author of GV Mobile+) seems to be saying the same thing:
https://twitter.com/seankovacs/status/603294454141825024

**Comment by Fofer** [comment]
Indeed.  <sigh>  This is a sad day.  GrowlVoice on the Mac, and GV Mobile+ on iPhone, were the only ways I enjoyed checking for my voicemail.  

Now Google's killed those off... is my only option to use Hangouts?

Yuck.

**Comment by Fofer** [comment]
In the meantime I have fired up my old Fluid (http://fluidapp.com) SSB standalone app, that pointed to this URL: https://www.google.com/voice/m?uipref=1#~voice:s=inbox&l=inbox
...and pinned it to my menu bar.  

It's ugly, but it works.

**Comment by Fofer** [comment]
Now Sean Kovacs (author of GV Mobile+) claims to have figured out a workaround:
https://twitter.com/seankovacs/status/603395739964801024

**Comment by itraynis** [comment]
I'm so ready for a workaround. Let's have it :)

**Comment by d235j** [comment]
Unfortunately the workaround would require lots of effort for a runtime patch :(

**Comment by Monkpie** [comment]
Perhaps it's fitting that the death of GrowlVoice has come so close to Memorial Day. Thank you for your efforts d235j. 

Just to summarize the alternatives that are currently available:

1. The fluid.app arrangement (mentioned by Fofer in this post).

2. The google voice extension for chrome
(https://chrome.google.com/webstore/detail/google-voice-by-google/kcnhkahnjcbndmmehfkdnkjomaanaooo?hl=en)

3. The google hangouts extension for chrome
(https://chrome.google.com/webstore/detail/hangouts/nckgahadagoaajjgafhacjanaoiihapd?hl=en)

If anyone else has a better idea, please post!

**Comment by Fofer** [comment]
Using any web browser to visit https://voice.google.com still works, too.

**Comment by hagure** [comment]
Ah, this is what I was afraid of……the day has finally come. 

RIP GrowlVoice ＿|￣|○

Going back to chrome extension & Fluid SSB I suppose.
